@here/harp-examples/src/textcanvas_dynamic.ts

```diff
@@ -25,8 +25,6 @@ import {
  * This example showcases how [[TextCanvas]] can handle dynamic loading of multiple [[FontCatalog]]
  * assets, as well as dynamic real-time text styling.
  *
- * For more information regarding basic [[TextCanvas]] initialization and usage, please check:
```

Comment:
why was this removed? TextCanvasMinimalExample still seems to exist.

Reply:
It is not currently available in harp.gl, but will be added back in the future, so I remove it until then